spacewalk/admin/spacewalk-admin.spec

```diff
@@ -86,6 +86,12 @@ ln -s spacewalk-service $RPM_BUILD_ROOT%{_sbindir}/rhn-satellite
 sed -i 's|#!/usr/bin/python|#!/usr/bin/python3|' $RPM_BUILD_ROOT/usr/bin/mgr-events-config.py
 %endif
 
+%post
```

Comment:
Why do we need this change?

Reply:
This is a fix which would be good to have already now. when we change .service files or add new, we need to reload the systemd daemon. Otherwise it does not know about the changes.

spacewalk-admin is not touched very often. So I think it is ok, to call a reload on every change.